wallet-api/src/main/java/org/exoplatform/wallet/reward/rest/RewardReportREST.java

```diff
@@ -127,4 +127,31 @@ public Response listRewards(@ApiParam(value = "limit of items to load", required
     }
   }
 
+  @GET
+  @Path("countRewards")
+  @RolesAllowed("users")
+  @Produces(MediaType.APPLICATION_JSON)
+  @ApiOperation(value = "Return sum of rewards for current user", httpMethod = "GET", produces = "application/json", response = Response.class, notes = "return sum of rewards per user")
+  @ApiResponses(value = {
+          @ApiResponse(code = HTTPStatus.OK, message = "Request fulfilled"),
+          @ApiResponse(code = HTTPStatus.UNAUTHORIZED, message = "Unauthorized operation"),
+          @ApiResponse(code = 500, message = "Internal server error") })
+  public Response countRewards() {
+    try {
+      Double sumRewards = rewardReportService.countRewards(WalletUtils.getCurrentUserId());
+      JSONObject result = new JSONObject();
+      result.put("sumRewards", sumRewards);
```

Comment:
Could you add Etag to this request to cache it on browser side 
an example https://github.com/meeds-io/social/blob/develop/component/service/src/main/java/org/exoplatform/social/rest/impl/identity/IdentityRestResourcesV1.java#L134